README.md

```diff
@@ -33,6 +33,45 @@ links to JIRA tickets will be appended at the bottom of the PR body.
 Optionally, the bot can be configured to automatically create a GitHub check listing Develocity build scans
 for every commit that has completed checks related to CI (GitHub Actions or Jenkins).
 
+### License agreement check
+
+The bot can be configured to check that the pull request body includes the license agreement text from the template. 
+To enable the check, add the following to the repository configuration:
```

Comment:
Maybe configuration needs to move to... the "Configuration" section? at least that's what we did for other categories.

Alternatively, I don't mind keeping your new format, but we should probably adapt pre-existing documentation to be consistent.